packages/flutter/lib/src/material/color_scheme.dart

```diff
@@ -4,167 +4,508 @@
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/widgets.dart';
+import 'package:material_color_utilities/material_color_utilities.dart';
 
 import 'colors.dart';
 import 'theme_data.dart';
 
-/// A set of twelve colors based on the
-/// [Material spec](https://material.io/design/color/the-color-system.html)
+/// A set of colors based on the
```

Comment:
We might as well admit how many colors there are here: a set of 25 colors...

It would be helpful to explain the Foo and onFoo naming conventions and to provide an overview of the colors.